CourseSketchProjects/IdentityServer/src/test/java/coursesketch/database/identity/IdentityManagerTest.java

```diff
@@ -0,0 +1,922 @@
+package coursesketch.database.identity;
+
+import com.coursesketch.test.utilities.AuthenticationHelper;
+import com.github.fakemongo.junit.FongoRule;
+import com.mongodb.BasicDBList;
+import com.mongodb.BasicDBObject;
+import com.mongodb.DB;
+import com.mongodb.DBCursor;
+import com.mongodb.DBObject;
+import coursesketch.database.auth.AuthenticationChecker;
+import coursesketch.database.auth.AuthenticationDataCreator;
+import coursesketch.database.auth.AuthenticationException;
+import coursesketch.database.auth.AuthenticationOptionChecker;
+import coursesketch.database.auth.Authenticator;
+import coursesketch.server.authentication.HashManager;
+import database.DatabaseAccessException;
+import database.DatabaseStringConstants;
+import org.bson.types.ObjectId;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.powermock.modules.junit4.PowerMockRunner;
+import protobuf.srl.school.School;
+import protobuf.srl.services.authentication.Authentication;
+
+import java.net.UnknownHostException;
+import java.nio.charset.StandardCharsets;
+import java.security.NoSuchAlgorithmException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import static database.DbSchoolUtility.getCollectionFromType;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyLong;
+import static org.mockito.Mockito.when;
+
+/**
+ * Created by dtracers on 10/8/2015.
+ */
+@RunWith(PowerMockRunner.class)
+public class IdentityManagerTest {
+
+    @Rule
+    public FongoRule fongo = new FongoRule();
+
+    private static final String VALID_USERNAME = "Valid username";
```

Comment:
why is it? its just a string for testing.

It does not matter what is in the string